bin/conjur-cli.rb

```diff
@@ -61,35 +61,36 @@ def test_select
     if account
       system "rake account:create[#{account}]" or exit $?.exitstatus
     end
-    
+
     if file_name = options[:file]
       raise "account option is required with file option" unless account
       system "rake policy:load[#{account},#{file_name}]" or exit $?.exitstatus
     end
-    
+
     exec "rails server -p #{options[:port]} -b #{options[:'bind-address']}"
   end
 end
 
 desc "Manage the policy"
 command :policy do |cgrp|
-  cgrp.desc "Load the policy from a file"
-  cgrp.arg_name "account file_name"
+  cgrp.desc "Load MAML policy from file(s)"
+  cgrp.arg :account
+  cgrp.arg_name "filename"
```

Comment:
You should remove this `arg_name` -- not only it doesn't make sense here, but it also disables strict argument handling.